pages/common/strip-nondeterminism.md

```diff
@@ -0,0 +1,15 @@
+# strip-nondeterminism
+
+> A tool to remove non-deterministic information (e.g. timestamps) from files.
+
+- Strip non-deterministic information from a file:
+
+`strip-nondeterminism {{file}}`
+
+- Strip nondeterministic information from a file while manually specifying the file type:
+
+`strip-nondeterminism --type {{ar|gzip|jar|zip}} {{file}}`
```

Comment:
If `ar|gzip|jar|zip` is not the only supported file types you should use a variable:
```suggestion
`strip-nondeterminism --type {{filetype}} {{path/to/file}}`
```

Reply:
I thought that they were the only ones, because the manpage only lists these four types. It turns out that there are indeed more: https://salsa.debian.org/reproducible-builds/strip-nondeterminism/-/blob/master/lib/File/StripNondeterminism.pm#L116